src/components/Tank/Tank.model.ts

```diff
@@ -0,0 +1,14 @@
+export declare type TankPosition = 'up' | 'down' | 'left' | 'right';
+export declare type TankColor = 'yellow' | 'white' | 'green' | 'red';
+export declare type TankType = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7';
+
+export interface TankProps {
+  color?: TankColor;
+  position?: TankPosition;
+  type?: TankType;
+}
+
+export interface TankCoordinate {
```

Comment:
- [ ] I don't know if this is the background-position or what but I think you can use our Position class in the Grid.

Reply:
**it is a coordinate of a whole tank in the sprite(and not related to CSS)** where [0;0] is the tank with specs `yellow`, `up`, `without-movement`(first-row), and [2;2] is a tank with specs `yellow`, `left`, `without-movement`(third-row)

Reply:
Oh. I kind of think that you can solve that in the CSS too. with just a few CSS variables.

Reply:
since our tank-configurations comes from logic(JS) - I think we don't have a CSS solution for it